src/libraries/System.Memory.Data/src/System.Memory.Data.csproj

```diff
@@ -28,6 +35,11 @@
 
   <ItemGroup Condition="'$(TargetFrameworkIdentifier)' != '.NETCoreApp'">
     <PackageReference Include="System.Memory" Version="$(SystemMemoryVersion)" />
-    <ProjectReference Include="$(LibrariesProjectRoot)System.Runtime.CompilerServices.Unsafe\src\System.Runtime.CompilerServices.Unsafe.ilproj" />
+    <!-- Not a direct dependency but reference to bring in the live asset instead of the transitive package dependency. -->
+    <ProjectReference Include="$(LibrariesProjectRoot)System.Runtime.CompilerServices.Unsafe\src\System.Runtime.CompilerServices.Unsafe.ilproj" PrivateAssets="all" />
```

Comment:
The comment here seems to disagree with `PrivateAssets`.  Don't we want this reference in the package?